docs/reference/content/driver/getting-started/quick-tour.md

```diff
@@ -1,418 +0,0 @@
-+++
```

Comment:
This is no longer needed and should have previously been removed as we have `quick-start.md`